pom.xml

```diff
@@ -56,9 +57,13 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.8</version>
+            <version>2.8.9</version>
+        </dependency>
+        <dependency>
+          <groupId>com.squareup.okhttp3</groupId>
+          <artifactId>okhttp</artifactId>
+          <version>${version.okhttp}</version>
```

Comment:
Make the dependency `<optional>true</optional>` to allow it to be included and used if the user also include the OKHttp library.

(I'm interested in creating one with Spring's WebClient as well, since I'll get success/failure and timing metrics)

Reply:
Thanks, good point :)

Reply:
done, I'm considering allowing you to configure the OkHttpClient directly, so you can setup an interceptor for prometheus metrics for the client